src/main/java/com/rultor/agents/github/qtn/QnReferredTo.java

```diff
@@ -82,20 +84,28 @@ public QnReferredTo(final String self, final Question qtn) {
     @Override
     public Req understand(final Comment.Smart comment,
         final URI home) throws IOException {
-        final String prefix = String.format("@%s ", this.login);
+        final String prefix = String.format("@%s", this.login);
         final Req req;
-        if (comment.body().trim().startsWith(prefix)) {
-            req = this.origin.understand(comment, home);
-        } else if (comment.body().contains(prefix)) {
-            new Answer(comment).post(
-                true,
-                String.format(
-                    QnReferredTo.PHRASES.getString("QnReferredTo.mentioned"),
-                    prefix
-                )
-            );
-            Logger.info(this, "mention found in #%d", comment.issue().number());
-            req = Req.DONE;
+        final Matcher matcher = Pattern.compile(
+            String.format(".*\\b?(%s\\b).*", prefix)
+        ).matcher(comment.body().trim());
+        if (matcher.matches()) {
+            if (matcher.start(1) == 0) {
+                req = this.origin.understand(comment, home);
+            } else {
+                new Answer(comment).post(
+                    true,
+                    String.format(
+                        QnReferredTo.PHRASES
+                            .getString("QnReferredTo.mentioned"),
```

Comment:
@original-brownbear please put `"QnReferredTo.mentioned"` to separate line and move `.getString` line above